### PR TITLE
Vibration is added to Every button for delight

### PR DESCRIPTION
**Pull request by Suryansh1720001**
Hello @AyushAgnihotri2025 
I added a slight vibration for the feeling of satisfaction. Please verify it and merge it accordingly.
Thank you

### CODE REVIEW

File: app/src/main/java/com/mrayush/calculator/MainActivity.kt [hunk 1]
```diff
@@ -85,6 +89,7 @@ class MainActivity : AppCompatActivity() {
         toggle.syncState()
 
         navigationView = findViewById<View>(R.id.navigation_menu) as NavigationView
+
```
Comment (AyushAgnihotri2025) [inline app/src/main/java/com/mrayush/calculator/MainActivity.kt]:
```suggestion
        
```

Kindly remove these kinds of unusual new lines.

**Comment by AyushAgnihotri2025** [comment]
Nice work @Suryansh1720001, 👍 . Kindly remove these kinds of unusual new lines so, that we can merge it.

**Comment by Suryansh1720001** [comment]
Hello @AyushAgnihotri2025,
I removed the extra line which you suggested. Now you can review it and merge it accordingly.
Thank you

**Review by AyushAgnihotri2025 — APPROVED** [review]
Nice work 👍 @Suryansh1720001, Keep It Up.

**Pull request merged**